python-idna (2.0-1) unstable; urgency=medium

  * Initial release. (closes: #756388)

 -- Tristan Seligmann <mithrandi@debian.org>  Mon, 08 Jun 2015 15:29:18 +0200
